devbox/provision/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore user-specific Pulumi configs
+Pulumi.*.yaml
```

Comment:
I think we're trying to reduce the number of gitignore files. Can you move this to the root gitignore file?
Since this is directory-specific you could do
`devbox/provision/Pulumi.*.yaml`